testsuite/integration/clustering/pom.xml

```diff
@@ -1499,6 +1555,67 @@
                                     </configurations>
                                 </configuration>
                             </execution>
+                            <!-- Provision a 'cloud-server' with 'singleton-ha'-->
+                            <execution>
+                                <id>cloud-server-singleton-ha-provisioning</id>
+                                <goals>
+                                    <goal>provision</goal>
+                                </goals>
+                                <phase>compile</phase>
+                                <configuration>
+                                    <install-dir>${project.build.directory}/wildfly-clustering-singleton-ha</install-dir>
+                                    <record-state>false</record-state>
+                                    <log-time>${galleon.log.time}</log-time>
+                                    <offline>${galleon.offline}</offline>
+                                    <plugin-options>
+                                        <jboss-maven-dist/>
+                                        <jboss-fork-embedded>${galleon.fork.embedded}</jboss-fork-embedded>
+                                        <optional-packages>passive+</optional-packages>
+                                    </plugin-options>
```

Comment:
N.B. This will impact https://github.com/wildfly/wildfly/pull/15730

Reply:
👍